### PR TITLE
lastEditedBy is set by add/edit, but not by physics packets

### PR DESCRIPTION
**Pull request by howard-stearns**
Because physics simulation might be done by anyone.
Fixes (part 1 of) https://highfidelity.fogbugz.com/f/cases/2728/

### CODE REVIEW

**Comment by howard-stearns** [comment]
test plan:

1. add an entity. Check in console that Entities.getEntityProperties(your-new-entity-id).lastEditedBy === MyAvatar.sessionUUID
2. in a new session, edit the same entity, Confirm that lastEditedBy now matches the new sessionUUID.
3. make the entity dynamic, and then start a third session. In the new session, move the entity around (with mouse grab or pointer grab). It should still have the previous lastEditedBy, not the new sessionUUID of the third session.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 6241889e38098d43fb9ef6058c558efdea7cd53b available at: https://gist.github.com/f9ce21775943d4fba1c26be8a30e7913<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9664-6241889.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9664-6241889.dmg

**Comment by davidkelly** [comment]
In step 3, _before_ I move the entity it correctly shows the last session uuid as lastEditedBy.  However, after I move it, it shows the current session ID.  Both client and server are running this PR -- odd...

File: libraries/entities/src/EntityTree.cpp [hunk 1]
```diff
@@ -1119,7 +1119,7 @@ int EntityTree::processEditPacketData(ReceivedMessage& message, const unsigned c
                     endLogging = usecTimestampNow();
 
                     startUpdate = usecTimestampNow();
-                    properties.setLastEditedBy(senderNode->getUUID());
+                    if (!isPhysics) properties.setLastEditedBy(senderNode->getUUID());
```
Comment (ZappoMan) [inline libraries/entities/src/EntityTree.cpp]: coding standard sez no!

should be:

```
 if (!isPhysics) {
    properties.setLastEditedBy(senderNode->getUUID());
}
```

Comment (howard-stearns) [inline libraries/entities/src/EntityTree.cpp]: fixed

**Comment by hifi-gustavo** [comment]
Build console output for commit: 487adc7f18af9bc25192c3445da7b12f51b4169b available at: https://gist.github.com/f9ce21775943d4fba1c26be8a30e7913<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9664-487adc7.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9664-487adc7.dmg

**Pull request merged**